turbinia/processors/mount_local.py

```diff
@@ -414,8 +496,18 @@ def PreprocessMountPartition(partition_path, filesystem_type):
   try:
     subprocess.check_call(mount_cmd)
   except subprocess.CalledProcessError as exception:
+    if filesystem_type == 'EXT':
+      # ext2 will not mount with the noload option, so this may be the cause of
+      # the error.
+      mount_cmd = ['sudo', 'mount', '-o', 'ro', partition_path, mount_path]
+      log.info('Mount failed, trying: {0:s}'.format(' '.join(mount_cmd)))
+      try:
+        subprocess.check_call(mount_cmd)
+      except subprocess.CalledProcessError as exception:
```

Comment:
Is this safe to do within the exception handler for the same exception type?

Reply:
It worked fine through all my testing, but it isn't very readable.
I've updated it.